.storybook/4.BREAKPOINTS.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Foundations / Breakpoints" />
+
+# Breakpoints
+
+Breakpoints are used for responsive layout
+
+## Screen Sizes
+
+There are 4 screen sizes that make up the breakpoints for the MetaMask extension
+
+- base: `0px`
+- sm: `576px`
```

Comment:
Retains current 576px size. Although I think we could change this in future or add the pop over size as `xs: 357px`